packages/desktop-client/src/components/App.tsx

```diff
@@ -33,7 +33,6 @@ import { DevelopmentTopBar } from './DevelopmentTopBar';
 import { FatalError } from './FatalError';
```

Comment:
The code in this PR is a bit messy.. My intention here is not to merge anything. But just share a very early draft of the concept of "synced prefs". Just to get early feedback before investing in developing it more.